test/language/statements/let/let-newline-await-in-normal-function.js

```diff
@@ -0,0 +1,20 @@
+// |reftest| error:SyntaxError
+// Copyright (C) 2017 Mozilla Corporation. All rights reserved.
```

Comment:
this file can be placed in the `test/language/statements/let/syntax` folder where we keep the tests for syntax errors.

Btw, you already authored other tests in there :) see `test/language/statements/let/syntax/let-let-declaration-split-across-two-lines.js`